docs/build/configure/_index.md

```diff
@@ -231,16 +231,31 @@ Find more information in the [processes documentation](/build/configure/processe
 
 ## Webhooks
 
-Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
-For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
+Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud, or when the internet connectivity of your machine changes.
+Viam provides two webhook types depending on the event you want to trigger on:
 
-Switch to **Raw JSON** mode to configure webhooks as follows:
+- `"part_data_ingested"`: trigger when a sensor on the machine part collects a new reading
+- `"part_online"`: trigger when the machine part is online
+
+One action you can trigger is sending a notification.
+For example, you can use the `"part_data_ingested"` type to configure a webhook to send you a notification when your robot's sensor collects a new reading, or use the `"part_online"` type to notify you when your machine part comes online.
```

Comment:
If you feel strongly that the example doesn't belong in the opening paragraph, here's a different option:
```suggestion
You can use webhooks to trigger any action executable by a cloud function.
For example, your webhook can trigger a function that sends you a phone notification.
```